gtest/spec/test_token_vendor.yaml

```diff
@@ -0,0 +1,85 @@
+title: token-vendor
+
+programs:
+  - id: 1
+    path: examples/target/wasm32-unknown-unknown/release/token_vendor.wasm
+    init_message:
+      kind: utf-8
+      # register ids
+      value: >
+        {
+          "members": [
+            "{6}",
+            "{7}"
+          ],
+          "reward": 10,
+          "code": "TEST",
+          "admins": [
+            "0100000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Why do not use just "1"? to increase readability
`let id: ProgramId = 1.into()`?

Reply:
because this contract has a real use and in my experience it is easier to specify the hex value of the id than the numerical interpretation.